src/cheri/cheri_isa_tables.adoc

```diff
@@ -145,19 +145,20 @@ include::insns/amoswap_32bit_cap.adoc[]
 <<<
 endif::[]
 
-[#rvy_h_insn_table, reftext="{rvy_h_ext_name}"]
-=== {rvy_h_ext_name}
-
-Specifying {cheri_base_ext_name} and "H" gives {rvy_h_ext_name} functionality, which adds virtualized capability load and store instructions.
-
-While HLVX.* only requires execute permission in the PTE, the authorizing CHERI capability must grant <<r_perm>>.
-
-.{rvy_h_ext_name} instruction extension
-[#{rvy_h_file_name}]
-[width="100%",options=header,cols="2,1,1,5",]
-|==============================================================================
-include::generated/{rvy_h_file_name}.adoc[]
-|==============================================================================
+//move to priv
```

Comment:
Could you move this to a priv tables file? And then we could also add MRET & SRET to a table?

Reply:
yeah ok